docs/essentials/haptic-feedback.md

```diff
@@ -1,16 +1,16 @@
 ---
 title: "Xamarin.Essentials: HapticFeedback"
-description: "TThis document describes the HapticFeedback class in Xamarin.Essentials, which lets you perform the haptic feedback."
+description: "This document describes the HapticFeedback class in Xamarin.Essentials, which lets you perform haptic feedback."
 ms.assetid: 7E8B24C4-2625-4DAE-A129-383542D34F1E
-author: dimonovdd
```

Comment:
@davidbritch 
Excuse me, but why was my name changed to another which is not me?